packages/backend-plugin-api/report.api.md

```diff
@@ -24,6 +24,40 @@ import { Readable } from 'stream';
 import type { Request as Request_2 } from 'express';
 import type { Response as Response_2 } from 'express';
 
+// @public (undocumented)
+export type AuditorCreateEvent<TRootMeta extends JsonObject> = (options: {
```

Comment:
No reason to define this as a function type is there? I think the following is more in line with our usual types structure:

```ts
export interface AuditorService {
  createEvent(options: AuditorServiceCreateEventOptions): ReturnType<AuditorServiceEvent>;
}
```

On top of that let's rename `AuditorEventSeverityLevel` to `AuditorServiceEventSeverityLevel`